carto_report/report.py

```diff
@@ -24,614 +25,595 @@
         
-            'real_storage':real_storage,
-            'used_storage':used_storage,
-            'pc_used':pc_used,
-            'left_storage':left_storage,
-            'pc_left':pc_left,
+        lds = lds.set_index('Service')
+        lds['Left'] = round(lds['Monthly Quota'] - lds['Used'],1)
+        lds['% Left'] = 100.00 - lds['% Used']
+        lds_df = lds[['Monthly Quota', 'Provider', 'Used', '% Used', 'Left', '% Left']]
+
+        return lds_df
+
+    ### get analysis and tables data
+
+    def getSizes(self, dsets_df):
+        '''
+        Method to get all tables sizes, know cartodbfied and non cartodbfied tables (analysis).
+        '''
+        
+        self.logger.info('Getting list of tables...')
         
-            'total_maps':total_maps,
-            'total_analysis':total_analysis,
-            'total_size_analysis':total_size_analysis,
-            'analysis_df': analysis_df,
-            'top_5_maps_date': top_5_maps_date,
+        all_tables = self.sql.send(
+                    "select pg_class.relname as name from pg_class, pg_roles, pg_namespace" +
+                    " where pg_roles.oid = pg_class.relowner and " +
+                    "pg_roles.rolname = current_user " +
+                    "and pg_namespace.oid = pg_class.relnamespace and pg_class.relkind = 'r'")['rows']
+
+        all_tables_df = json_normalize(all_tables)
         
-            'sync': sync,
-            'total_dsets':total_dsets,
-            'total_size_tbls':total_size_tbls,
-            'private':private,
-            'link':link,
-            'public':public,
-            'geo':geo,
-            'points':points,
-            'lines':lines,
-            'polys':polys,
-            'none_tbls':none_tbls,
-            'top_5_dsets_size': top_5_dsets_size,
-            'top_5_dsets_date': top_5_dsets_date,
         
-            'lds': lds,
+        self.logger.info('Retrieved {} tables.'.format(len(all_tables_df)))
         
-            'html_fig1': fig_to_html(fig1),
-            'html_fig2': fig_to_html(fig2)
-        })
+        dsets_df['cartodbfied'] = 'Yes'
+        all_tables_df = all_tables_df.merge(dsets_df, on='name', how='left')
+        all_tables_df['cartodbfied'] = all_tables_df['cartodbfied'].fillna('No')
+        all_tables_df['size'] = 0
+        
+        self.logger.info('Getting table sizes...')
+        
+        for index, row in all_tables_df.iterrows():
+            try:
+                size = self.sql.send("select pg_total_relation_size('" + row['name'] + "') as size")['rows'][0].get('size')
+            except:
+                self.logger.info('Error at: ' + str(row['name']))
+            
+            all_tables_df.set_value(index,'size',size)
+            
+        self.logger.info('Table sizes retrieved with a sum of {} MB'.format(all_tables_df['size'].sum()))
+            
+        return all_tables_df
+
+    ### get analysis names table
+
+    def getAnalysisNames(self, all_tables_df):
```

Comment:
This is misleading, it should be more clear in the report and the code that we are always talking about **cached analysis tables**, so the maps in the user account may have many more analyses applied, but they are computed on the fly and this only  reflects those of them that are saved in the account by the maps api.

Reply:
So what you are saying is changing the naming (function and output)?

Reply:
Changing the titles in the report and the name for the method that reports about analysis

Reply:
Done.